ui/packages/components/src/Select/Select.tsx

```diff
@@ -44,12 +45,14 @@ export function Select({
     <Listbox value={defaultValue} onChange={onChange} multiple={multiple}>
```

Comment:
Is it correct to set `value` to `defaultValue`? When I'm using a component, I think the `defaultValue` prop will initially set the value and then from there the component is in charge of internally managing state. Should we rename our `defaultValue` prop to `value` to reflect that?